Guides/UPI/vSphere_terraform/main.tf

```diff
@@ -41,7 +41,7 @@ module "bootstrap" {
   memory               = "8192"
   num_cpu              = "4"
   num_cores_per_socket = "4"
-  disk_size            = "60"
+  disk_size            = "128"
```

Comment:
Lets align it with [the docs](https://docs.okd.io/latest/installing/installing_vsphere/installing-restricted-networks-vsphere.html#minimum-resource-requirements_installing-restricted-networks-vsphere) and set it to `120`?

Reply:
Done